src/events/cd-event-ticket.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="cd-event-tickets" :class="{ 'before-disabled': notAttending }">
-    <div class="cd-event-tickets__head" :class="{ 'head-disabled': notAttending }"></div>
+  <div class="cd-event-tickets" :class="{ 'cd-event-tickets--disabled': notAttending }">
+    <div class="cd-event-tickets__head" :class="{ 'cd-event-tickets__head--disabled': notAttending }"></div>
```

Comment:
This dynamic class isn't needed anymore. The `--disabled` on the root element is what changes its styles now.